indico/modules/events/registration/client/js/form/fields/CountryInput.jsx

```diff
@@ -20,13 +20,23 @@ import {getStaticData} from '../selectors';
 const isoToFlag = country =>
   String.fromCodePoint(...country.split('').map(c => c.charCodeAt() + 0x1f1a5));
 
+const asciiSearch = (options, query) => {
```

Comment:
I'd add either a comment to the commit message or a docstring explaining a bit what this does together with an example e.g. `Türkiye` -> `Turkiye`. (You could even include the link to the SO answer for reference)